Lesson17/Merve/script.js

```diff
@@ -0,0 +1,156 @@
+// 1. Write a loop that prints out the numbers from 1 to 10.
+for (let i = 1; i <= 10; i++) {
+    console.log(i);
+}
+console.log('- - - - - - - - - - - - - - - - - ');
+// 2. Write a loop that prints out the odd numbers from 1 to 20.
+for (let i = 1; i <= 20; i++) {
```

Comment:
This one does not work correctly as it prints all numbers from 1 to 20, not only odd numbers
```suggestion
for (let i = 1; i <= 20; i += 2) {
```